inc/tag_replacer.php

```diff
@@ -62,22 +62,106 @@ public function init() {
 	 *
 	 * @return mixed
 	 */
+
+	/**
+	 *  Replace in given content the given image tags with video tags is image is gif
+	 *
+	 * @param string $image_url Image url to process.
+	 * @param string $image_tag The tag to replace.
+	 * @param string $content The content to process.
+	 *
+	 * @return bool
+	 */
+	public function img_to_video( $image_url, $image_tag, &$content ) {
+		if ( $this->settings->get( 'img_to_video' ) === 'disabled' ) {
+			return false;
+		}
+
+		if ( $this->is_valid_mimetype_from_url( $image_url, [ 'gif' => false ] ) ) {
+			return false;
+		}
+
+		$link_webp = apply_filters(
```

Comment:
We don't seem to use this anywhere.

Reply:
it was a link in the video tag, it looks like it got removed from there i think we should put it back there